sequins.go

```diff
@@ -132,14 +132,26 @@ func (s *sequins) initCluster() error {
 	if err != nil {
 		return err
 	}
+	
+	ip := ""
+	ipAddresses, err := net.LookupHost(hostname)
+	if err	!= nil {
```

Comment:
Err, is that a tab or something? Something looks off with the formatting

(Do we validate that sequins code is gofmt'd?)